mirrord/layer/src/socket/ops.rs

```diff
@@ -1135,6 +1135,13 @@ pub(super) fn send_to(
         .remove(&sockfd)
         .ok_or(Bypass::LocalFdNotFound(sockfd))?;
 
+    // we don't support unix sockets which don't use `connect`
+    if (destination.is_unix() || user_socket_info.domain == AF_UNIX)
+        && user_socket_info.user_socket_info.state != SocketState::Connected
+    {
+        return Bypass::UnixSocket(destination);
```

Comment:
Doesn't make a difference right now, but I think the intended bypass variant for this case is `Domain`, not `UnixSocket`, according to their respective documentations.